presto-verifier/src/main/java/com/facebook/presto/verifier/rewrite/VerificationQueryRewriterFactory.java

```diff
@@ -50,19 +51,23 @@
     private final List<Property> controlTableProperties;
     private final List<Property> testTableProperties;
 
+    private final boolean rewriteNonDeterministicColumns;
+
     @Inject
     public VerificationQueryRewriterFactory(
             SqlParser sqlParser,
             TypeManager typeManager,
             @ForControl QueryRewriteConfig controlConfig,
-            @ForTest QueryRewriteConfig testConfig)
+            @ForTest QueryRewriteConfig testConfig,
+            VerifierConfig verifierConfig)
```

Comment:
Why don't we put the new config into `QueryRewriteConfig`?

Reply:
Makes sense. Specifying the mapping twice would be not good.